### PR TITLE
Ty/emails as text

### PR DESCRIPTION
**Pull request by AlephNotation**
stop rendering emails as links to stop models from clicking on them and opening the system email client

### CODE REVIEW

File: src/browser/page.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ import { fetchMemorySequence, remember } from "../collectiveMemory/remember";
 import { ModelResponseType } from "../types";
 import { DEFAULT_STATE_ACTION_PAIRS } from "../collectiveMemory/examples";
 import { Memory } from "../types/memory.types";
+import { isEmail } from "../utils/airaUtils";
```
Comment (matildepark) [inline src/browser/page.ts:36]: you mean, like, aria?

**Comment by matildepark** [comment]
Seems okay, just kinda curious if we could've just added some interception of the `mailto:` schema to prevent nav. Or really any schema that isn't `https://` or `http://` if there's IPFS links or ... Gopher or whatever.

File: src/browser/page.ts (hunk 1 above)
Comment (AlephNotation) [reply src/browser/page.ts]: deleted this file

File: src/browser/browser.ts [hunk 2]
```diff
@@ -129,6 +129,16 @@ export class Browser {
     disableMemory?: boolean;
   }): Promise<Page> {
     const basePage = await this.browser.newPage();
+
+    await basePage.setRequestInterception(true);
+    basePage.on("request", (request) => {
+      if (request.url().startsWith("mailto:")) {
+        request.abort();
```
Comment (matildepark) [inline src/browser/browser.ts:136]: my second question is, how does this interact with the state machine? will it just keep hammering the link?

Comment (AlephNotation) [reply src/browser/browser.ts:136]: say more please

**Review by matildepark — APPROVED** [review]

**Pull request merged**